src/platform/lunarlake/include/platform/drivers/alh.h

```diff
@@ -0,0 +1,19 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2022 Intel Corporation. All rights reserved.
+ */
+
+#ifdef __SOF_DRIVERS_ALH__
+
+#ifndef __PLATFORM_DRIVERS_ALH__
+#define __PLATFORM_DRIVERS_ALH__
```

Comment:
if this is platform specific, suggest add LNL in the definition in case of other platform duplicated definition.
__LNL_PLATFORM_DRIVERS_ALH__